packages/commons/src/events/ActionInput.ts

```diff
@@ -112,7 +113,7 @@ export type ArchiveActionInput = z.infer<typeof ArchiveActionInput>
 export const AssignActionInput = BaseActionInput.merge(
   z.object({
     type: z.literal(ActionType.ASSIGN).default(ActionType.ASSIGN),
-    assignedTo: z.string()
+    assignedTo: z.string() // @TODO: Could this be set to UUID?
```

Comment:
I think the assignedTo is a user id, which are not uuids

so I'm quite sure the answer to the comment is no